modules/op-release-notes-1-4.adoc

```diff
@@ -106,7 +106,12 @@ fsGroup:
 
 * The `creds-init` helper image for building and deploying is removed.
 
-* In the triggers spec and binding, the deprecated field `template.Name` is removed in favor of `template.Ref`.
+* In the triggers spec and binding, the deprecated field `template.name` is removed in favor of `template.ref`. You should update all `eventListener` definitions to use the `ref` field.
++
+[NOTE]
+====
+Upgrades from {pipelines-shortname} 1.3.x and earlier versions to {pipelines-shortname} 1.4.0 breaks event listeners because of the unavailability of the `template.name` field. For such cases, use {pipelines-shortname} 1.4.1 to avail the `template.name` field which has been restored.
```

Comment:
breaks/break
Suggest: to avail the restored `template.name` field.

Reply:
Changed the start of the sentence to `Upgrade`, so that `breaks` becomes grammatically correct.